org.emptyflow.ArdorQuery.yml

```diff
@@ -0,0 +1,26 @@
+app-id: org.emptyflow.ArdorQuery
```

Comment:
emptyflow.org doesn't seem to resolve.

Reply:
The domain must be under the control of the project developers. `emptyflow.org` doesn't exist. This is a hard requirement.

Reply:
Ok, I buy domain, now it can be resolved.